README.md

```diff
@@ -14,34 +16,57 @@ Windows users may have trouble with the default installation folder of pnpm:
 
 Documentation for installing pnpm https://pnpm.io/installation.
 
+# apps & packages
+
+- **apps/web**  
+  main nextjs web app with backend routes
+
+- **apps/web-e2e**  
+  browser tests build with playwright
+
+- **packages/backend**  
+  the GraphQL backend with database build with prisma and pothos
+
+- **packages/pulumi**  
+  the infrastructure as code package build with pulumi
+
+- **packages/ui**  
+  re-usable UI components
+
+- **packages/tsconfig**  
+  central place to store all tsconfigs for all packages that use tsc
+
 # Getting Started
 
-The main application is located in the app folder as next.js app including backend and frontend.
+The main application is located in the apps/web folder as next.js typescript application. It can be started from the root folder using:
 
 ```bash
-cd app
+pnpm run dev
 ```
 
+Make sure the app runs on the right port your SSO is configured for, usually port 3000.
+
 ## Start the database server
 
 The database server has to be started prior to make the backend work.
 
 ```bash
-docker-compose up
+docker-compose up -d
 ```
 
 This app is developed using prisma.io. You can start the prisma studio to view/modify data by using:
 
 ```bash
-npx prisma studio
+cd packages/backend
+pnpx prisma studio
```

Comment:
I think this won't work with our single `.env` file in the root folder.

What about if we provide the `prisma.schema` location in the root `package.json`? I tried it and it worked for me.

Reply:
I added a turbo script
```bash
pnpm prisma-studio
```

This can be executed without having to go into the backend directory first.